WebDevelopment.html

```diff
@@ -0,0 +1,59 @@
+    </head>
+    <body>
+        <h1>WEB DEVELOPMENT</h1>
+        <img src="./image/Web Develoipment.png" alt="">
+        <H2>DESCRIPTION</H2>
+        <p>
+            <H2>SKILLS</H2>
+            <strong>Communication</strong>
+        </p>
+        <P>
+            Whether you’re part of an extensive engineering department or going solo as a contractor, you’ll need to communicate to
+            stakeholders or team members throughout your projects. Remain open to ideas from designers, clients, and others on your
+            team, and speak up if you anticipate roadblocks. Promoting open lines of communication will lead to a better finished
+            product, and help you improve as a developer.
+        </P>
+        <p>
+            <strong>Creativity</strong>
+        </p>
+        <P>
+            Developers are responsible for finding the most elegant solution for the end-user. As you take on more complex projects,
+            you’ll find that there is rarely one answer to a problem: you may have multiple avenues at your disposal, and it takes a
+            creative mind to imagine how they’ll each perform.
+        </P>
+        <p>
+            <strong>Logical thinking</strong>
+        </p>
+        <b>
+            Programming languages operate on rules and definitions. If you’re naturally skilled at thinking through problems
+            logically, you’ll pick up these languages much more quickly. As you see them in action, you’ll develop a knack for
+            breaking down statements into their core parts and predicting logical outcomes.
+            </P>
+            <p>
+                <strong>Attention to detail</strong>
+            </p>
+            <P>
+                Even the most minute typo can lead to a 404. Programmers learn to re-read their code multiple times as they check for
+                errors and look for efficiencies. It takes patience, laser-focus, and an ability to look at your own work with an
+                objective eye.
+            </P>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
+            <a
+                href="https://www.pluralsight.com/browse/software-development/web-development?aid=7010a000002LUv2AAG&promo=&utm_source=non_branded&utm_medium=digital_paid_search_google&utm_campaign=XYZ_EMEA_Dynamic&utm_content=&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlVbq6p47uycFayj344Jb0C5jtAspXjIh8rmKych886hOvVMqj1H-7IaAjTzEALw_wcB">Getting
+                GROW YOUR WEB DEVELOPMENT SKILLSUnderline</a>
+            <br>
+            <a href="https://www.udemy.com/?utm_source=adwords&utm_medium=udemyads&utm_campaign=Generic-Broad_la.EN_cc.ROW&utm_content=deal4584&utm_term=_._ag_86841142536_._ad_405920124403_._kw_learn%20online_._de_c_._dm__._pl__._ti_kwd-11046951_._li_1010294_._pd__._&matchtype=b&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlXKIheYDCz6aR5iiWD7SFFoDvpdnXGPDfeXxCZ-1zNOlZrEtSDWvzAaAnSXEALw_wcB">Web Development
```

Comment:
The URL is too long and also contains info you don't need
```suggestion
            <a href="https://www.udemy.com">Web Development
```